### DISCUSSION TITLE
Arctic Code Vault Contributor

### DISCUSSION THREAD

**Discussion by jorgesandovalm** [community/community#18711] [Other Feature Feedback, Questions, & Ideas] [answered]
Could you guide me about github achievements, which ones currently exist and how can I get them? Almost all of my friends have "Github Archive Program" but I can't find any information about that.

**Comment by thomscoder** [comment, marked as answer]
You can get Github achievements by being active in the Github/OSS community by contributing to repositories, replying in discussions and so on. Basically the more active you are the higher the chances to get one
Currently there are: 
- YOLO (merging a commit without a code review)
- PULL SHARK (when your pull requests get merged)
- GALAXY BRAIN (when answering in discussions)
- PUBLIC SPONSOR (when sponsoring someone) 
- [ARCTIC CODE VAULT](https://archiveprogram.github.com/)
and probably some others I'm missing...

You can read more about that [here](https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/)

**Reply by SebastienBtr** [reply]
That'd be nice to be able to access the list of existing achievements somewhere, and potentially your progress on a given badge (assuming some achievements will require x number of actions)

**Reply by thomscoder** [reply]
> That'd be nice to be able to access the list of existing achievements somewhere, and potentially your progress on a given badge (assuming some achievements will require x number of actions)

You can see a list here: https://github.com/Schweinepriester/github-profile-achievements and the thresolds to reach certain tiers

**Comment by JoshuaLuckers** [comment]
If you click on an achievement it shows more details about it. 

<img width="486" alt="Screen Shot 2022-06-16 at 11 24 10" src="https://user-images.githubusercontent.com/1268085/174038703-d945a453-73a1-496e-8934-fd6eda3124f5.png">
